.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
@martinmacko47 V nastaveniach treba nastavit [protected branch](https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks) (urcite to mas, len ja ku /settings nemam pristup)

Reply:
Aktualne mame taketo nastavenia protected branchi:

![protected](https://user-images.githubusercontent.com/5547275/84590536-dba9e400-ae37-11ea-9104-fd24279860e5.png)

PR robime iba do `master`. Brancha `deploy` sluzi iba ako pointer, co je prave nasadene na produkcii, nerobime do nej priamo ziadne commtiy ani PR.